Yasgui/packages/yasqe/src/index.ts

```diff
@@ -945,10 +954,29 @@ export class Yasqe extends CodeMirror {
   /**
    * Querying
    */
-  public query(config?: Sparql.YasqeAjaxConfig) {
+  public query(config?: Sparql.YasqeAjaxConfig, explain = false) {
```

Comment:
`explain` is not exactly conventional name for a boolean variable. Would you consider renaming it to `isExplainQuery` or something similar?